src/Minigames/Reaction_Contest_UI.py

```diff
@@ -74,7 +77,7 @@ async def _play(self) -> str:
             await asyncio.sleep(0.1)
 
         winner_index = self._game.get_winner()
-        winner = self._players[winner_index]
+        winner = "bot" if winner_index == 1 else self._players[winner_index]
```

Comment:
Currently "bot" is the winner if 2 players play and the second player wins. The second player can't win. Or am I confused?